### PR TITLE
Refeactor the repo manager

### PR DESCRIPTION
**Pull request by shiyu22**
Signed-off-by: shiyu22 <shiyu.chen@zilliz.com>

### CODE REVIEW

File: towhee/hub/bin/hub_tools.py [hunk 1]
```diff
@@ -14,73 +14,97 @@
...
-    elif args.type == 'nnoperator':
-        manager = OperatorManager(args.author, args.repo)
-    elif args.type == 'pipeline':
-        manager = PipelineManager(args.author, args.repo)
-    return manager
+def get_repo_obj(args):
+    if args.command == 'download':
+        repo_obj = RepoManager(args.author, args.repo)
+    elif args.command == 'clone':
+        repo_obj = GitUtils(args.author, args.repo)
```
Comment (junjiejiangjjj) [inline towhee/hub/bin/hub_tools.py:30]: Why return `GitUtils`, and others are `Manger`.

Comment (shiyu22) [reply towhee/hub/bin/hub_tools.py:30]: There are `create/init/generate-yaml/download/clone` methonds, and `clone` is for GitUtils, others for Manager.

**Review by Chiiizzzy — APPROVED** [review]

File: towhee/utils/hub_utils.py [hunk 2]
```diff
@@ -12,200 +12,267 @@
...
+
+        Returns:
+            (`Response`)
+                A file content in the file_path.
+
+        Raises:
+            (`HTTPError`)
+                Raise error in request.
+        """
+        url = f'{self._root}/{self._author}/{self._repo}/media/branch/{tag}/{file_path}'
```
[the remark this answers is not in the document]
Comment (shiyu22) [reply towhee/utils/hub_utils.py:148]: To get the lfs file.

**Pull request merged**